lib/router/router.js

```diff
@@ -153,13 +153,14 @@ export default class Router {
       return true
     }
 
+    const { asPathname, asQuery } = parse(as, true)
```

Comment:
doesn’t this need to be `const { pathname: asPathname, query: asQuery } = parse(as, true)`?

Reply:
Woo, yes. 🤦‍♂️  Let me double check that this is working correctly.

Reply:
Yeah looks good. Patched.